types/index.d.ts

```diff
@@ -40,15 +42,8 @@ type ScaleFunctions<T> = {
 
 declare const Zoom: Plugin & {
   zoomFunctions: ScaleFunctions<ZoomFunction>;
+  zoomRectFunctions: ScaleFunctions<ZoomRectFunction>;
   panFunctions: ScaleFunctions<PanFunction>;
 };
 
 export default Zoom;
-
-export function pan(chart: Chart, amount: PanAmount, scales?: Scale[], mode?: UpdateMode): void;
-export function zoom(chart: Chart, amount: ZoomAmount, mode?: UpdateMode): void;
-export function zoomScale(chart: Chart, scaleId: string, range: ScaleRange, mode?: UpdateMode): void;
-export function resetZoom(chart: Chart, mode?: UpdateMode): void;
-export function getZoomLevel(chart: Chart): number;
-export function getInitialScaleBounds(chart: Chart): Record<string, {min: number, max: number}>;
-export function isZoomedOrPanned(chart: Chart): boolean;
```

Comment:
aren't these available anymore?

Reply:
I made a mistake; I saw that they weren't available in `index.js` (the non-ESM build), but I missed that they are available in `index.esm.js` (the ESM build). And, in some local testing, I was somehow using a non-ESM setup, so the functions weren't available.

I restored these.